ddtrace/tracer/option.go

```diff
@@ -190,6 +190,9 @@ func newConfig(opts ...StartOption) *config {
 	if v := os.Getenv("DD_TRACE_SOURCE_HOSTNAME"); v != "" {
 		c.hostname = v
 	}
+	if v := os.Getenv("DD_AGENT_HOST"); v != "" {
+		c.agentAddr = v
+	}
```

Comment:
Actually, looking at the code more, what do you think if we would instead modify this line at the top of `newConfig`:
```go
	c.agentAddr = defaultAddress
```
to become:
```go
	c.agentAddr = resolveAddr(defaultAddress)
```

`resolveAddr` is in `transport.go` and should likely be renamed more accurately to maybe `resolveAgentAddr`.

Reply:
Great suggestion. I've made that change and renamed `resolveAddr` to `resolveAgentAddr`.